scalafix-tests/unit/src/test/scala/scalafix/tests/util/IntervalSetTest.scala

```diff
@@ -5,24 +5,27 @@ import org.scalactic.source.Position
 
 import scalafix.internal.util.IntervalSet
 
-class IntervalSetSpec() extends FunSuite {
-  // format: off
-  test("contains") {    
+class IntervalSetTest() extends FunSuite {
+
+  test("contains") {
     val set = IntervalSet((1, 2), (4, 5))
     assert(!set.contains(0))
-    assert( set.contains(1))
-    assert( set.contains(2))
+    assert(set.contains(1))
+    assert(set.contains(2))
     assert(!set.contains(3))
-    assert( set.contains(4))
-    assert( set.contains(5))
+    assert(set.contains(4))
+    assert(set.contains(5))
     assert(!set.contains(6))
   }
 
-  test("intersect") {
+  test("intersects") {
     val set = IntervalSet((1, 2), (5, 8))
-    def in(start: Int, end: Int)(implicit pos: Position): Unit  = assert( set.intersect(start, end))
-    def out(start: Int, end: Int)(implicit pos: Position): Unit = assert(!set.intersect(start, end))
+    def in(start: Int, end: Int)(implicit pos: Position): Unit =
+      assert(set.intersects(start, end))
+    def out(start: Int, end: Int)(implicit pos: Position): Unit =
+      assert(!set.intersects(start, end))
 
+    // format: off
```

Comment:
You can avoid this by padding more spaces before the comment
```
//                  //    0 1 2 3 4 5 6 7 8 9 
```

But it's not necessary.